cfpb/ccdb5-ui
# Various Bug Fixes (DATAP-1570 and DATAP-1575)

Various Bug Fixes
- Hide buttons from print view 
- Align charts in print view with associated legend
- Increase margins on all sides for warning and error notifications

## Additions

-

## Removals

-

## Changes

-

## Testing
- Run search query [here](https://www.consumerfinance.gov/data-research/consumer-complaints/search/?chartType=line&dateInterval=Month&dateRange=All&date_received_max=2024-10-16&date_received_min=2011-12-01&focus=Credit%20reporting%2C%20credit%20repair%20services%2C%20or%20other%20personal%20consumer%20reports&has_narrative=true&lens=Product&product=Credit%20reporting%2C%20credit%20repair%20services%2C%20or%20other%20personal%20consumer%20reports&product=Credit%20reporting%2C%20credit%20repair%20services%2C%20or%20other%20personal%20consumer%20reports%E2%80%A2Credit%20reporting&product=Credit%20reporting%2C%20credit%20repair%20services%2C%20or%20other%20personal%20consumer%20reports%E2%80%A2Credit%20repair%20services&product=Credit%20reporting%2C%20credit%20repair%20services%2C%20or%20other%20personal%20consumer%20reports%E2%80%A2Other%20personal%20consumer%20report&searchField=complaint_what_happened&searchText=money&subLens=sub_product&tab=Trends)
- Press CMD + P
- Print screen should match screenshots
- Run warning query
- Warning notification should match screenshot

## Screenshots
![Screenshot 2024-10-30 at 3 07 01 PM](https://github.com/user-attachments/assets/0b1e1cdd-8cf4-4346-a6f6-d4dd2bc05222)

![Screenshot 2024-10-30 at 3 07 14 PM](https://github.com/user-attachments/assets/efdcf859-ecff-49e6-bee8-d437013fbd57)

![Screenshot 2024-10-30 at 3 14 33 PM](https://github.com/user-attachments/assets/aebd1230-dbef-4a3c-8fd1-4c6c386f8c98)


## Notes



-

## Todos

-

## Checklist
